packages/@webex/plugin-meetings/src/common/browser-detection.ts

```diff
@@ -1,3 +1,5 @@
+// @ts-nocheck
```

Comment:
Question: I see "@ts-nocheck" being used in all the files, so does that mean that even if we use the specific types somewhere (instead of `any`) then all ts errors are ignored anyway, so we won't see that we're using the types incorrectly?

Reply:
Correct @marcin-bazyl , We can't fix/add all the types at once, it will be done gradually. As part of this PR we will have `.ts` files, types picked from jsDoc and `any` at rest of the places. Once this PR is merged we will keep adding types as we touch a file and remove `//@ts-nocheck`.

We will also need to clean function param types - Currently `ts-migrate` picked type from jsDoc and added it as a raw type everywhere but at many places param types are kind of object which could be converted to interfaces.

Reply:
@rajeshtezu90 you are right you can not do all the migration in one step, but still not really make sense to add `// @ts-nocheck` to each files. TS can handle mixed source very well, so you should keep the `js` extension and change it to TS only when you are ready to fix the types as well